WORKSPACE

```diff
@@ -98,8 +57,52 @@ maven_jar(
   name = "com_squareup_okio_okio_1_6_0",
   artifact = "com.squareup.okio:okio:jar:1.6.0"
 )
-http_archive(
-    name = "com_google_protobuf",
-    urls = ["https://github.com/google/protobuf/archive/3.5.1.1.zip"],
-    strip_prefix = "protobuf-3.5.1.1",
+
+git_repository(
+    name = 'temp',
+    remote = 'https://github.com/vmax/temp',
```

Comment:
Looks like you're still using your repo..

Reply:
@oferb please downgrade plugin version in `startupos_binaries` so I could use those